### DISCUSSION TITLE
how to search tables across multiple database connections

### DISCUSSION THREAD

**Discussion by ct-groups** [dbeaver/dbeaver#14645] [General]
how to search tables across multiple database connections,and automatically filters other connections

**Comment by LonwoLonwo** [comment]
Hello @rainman-code 

You can use Database Navigator Filter https://github.com/dbeaver/dbeaver/wiki/Filter-Database-Objects

![2021-11-23 10_45_51-Window](https://user-images.githubusercontent.com/45152336/142987520-d28714e2-00e9-46fd-b96f-64c0b2b6d063.png)

**Reply by LonwoLonwo** [reply]
> Is it possible to search a table quikly, across multiple database connections without directiory tree expanded?

Unfortunately, DBeaver does not have such functionality. Because of its potential heaviness.

You can use Metadata Search as well as the Navigator Tree Table Filter. But Metadata Search also works only on one connection.

![2023-01-31 11_23_35-DBeaver Ultimate 22 3 3 - 11 24 324FFd](https://user-images.githubusercontent.com/45152336/215707210-7b04ab7a-588f-44d9-a1cc-36f6d9a3dd54.png)
